var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -15,6 +15,7 @@ class Libxc(AutotoolsPackage, CudaPackage):
 
     license("MPL-2.0-no-copyleft-exception")
 
+    version("7.0.0", sha256="8d4e343041c9cd869833822f57744872076ae709a613c118d70605539fb13a77")
     version("6.2.2", sha256="d1b65ef74615a1e539d87a0e6662f04baf3a2316706b4e2e686da3193b26b20f")
```

Comment:
@prckent, @lang-m, here is a suggestion that might fix CI by not making 7.0.0 the preferred version yet but instead keeping 6.2.2 as the preferred version:
```suggestion
    version(
        "6.2.2",
        sha256="d1b65ef74615a1e539d87a0e6662f04baf3a2316706b4e2e686da3193b26b20f")
        preferred=True,
    )
```
With this, recipes that need `libxc@7:` can already depend on it, and recipes can also be updated to declare which versions need to depend on `libxc@:6` can add those version ranges too.

This way of integrating libxc-7.0.0 allows testing all users of `libxc` with the new version (e.g. by forcing the build with `recipe ^libxc@7`, but the default builds (in CI and outside of CI) are not affected by adding libxc-7.0.0.

Reply:
Thanks for the suggestion, but I think the preferred version actually should be the latest, and pinning the version is not actually preferred. Based on the release notes (1) v7 includes fixes for physically correct results for some uses (2) only programs depending on certain parts of the Fortran API in certain ways will need an update. Packages using C and Python interfaces will likely be fine and need no update. In this case I think most packages will not need any update at all. Any that do can consider implementing their own policy that best suits them.